migrations/versions/426f1ad50ca2_.py

```diff
@@ -0,0 +1,34 @@
+"""empty message
+
+Revision ID: 426f1ad50ca2
+Revises: 598562686e8
+Create Date: 2015-02-18 15:46:44.779144
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '426f1ad50ca2'
+down_revision = '598562686e8'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('items_users',
```

Comment:
We call that table role_users. And it makes sense because there are more users than roles. It's a Western convention but I like the image of the shorter column being on the left side.

Similarly, we're likely to have less users than columns, so I think we should keep that convention.

Thoughts?